scripts/run_tests

```diff
@@ -13,6 +13,10 @@ cargo clippy --all-targets --manifest-path=./sdk/rust/Cargo.toml -- --deny=warni
 cargo test --all-targets --manifest-path=./examples/Cargo.toml
 cargo clippy --all-targets --manifest-path=./examples/Cargo.toml -- --deny=warnings
 
+# Check everything builds (except Asylo-specific targets).
+bazel build -k "${bazel_build_flags[@]}" -- //oak/...:all -//oak/server/asylo:all
```

Comment:
Do we need to also compile `//oak/server/asylo:oak` with an Asylo toolchain (`"${bazel_build_flags[@]}" --config=enc-sim`) ? Or it's already checked during `./scripts/build_server_asylo` in GCB.

Reply:
That target should get caught by the main `build_server_asylo` step as you say.

(I *think* that the only code that is un-buildable on the host system should live in the `asylo/` subdir, so this step should catch any new directories or build targets under `//oak` that don't get a build step elsewhere.)